Week3/homework/step2-7.js

```diff
@@ -1,15 +1,18 @@
 'use strict';
 
+// In this first example, we see that this function passes directly the variable value inside the function.
+// Changing the argument inside the function doesn't affect the variable passed outside of the function.
+// So as long as we don't console log the variable inside the function (f1) our variables 'true' value will remain as 9.
```

Comment:
While this is true, it's only part of the story. 
It's also important to understand the difference between changing the value of a variable (like changing x from 9 to 10), and changing the a property of a variable (like changing val.x from 9 to 10). Have a read through some of the discussions on this page https://stackoverflow.com/questions/6605640/javascript-by-reference-vs-by-value